src/chocolatey/infrastructure.app/services/TemplateService.cs

```diff
@@ -348,7 +351,13 @@ protected void ListCustomTemplateInformation(ChocolateyConfiguration configurati
             }
             else
             {
-                this.Log().Info("{0}|{1}".FormatWith(configuration.TemplateCommand.Name, pkgVersion));
+                if (configuration.IncludeHeaders && !_hasHeaderRowBeenOutput)

```

Comment:
Same question as before. Any reason why this hasn't been combined as a `else if` statement?

Reply:
Same comment as before, this is needed to only output header row when required.